### PR TITLE
Update docs for clarity and consistency

### PR DESCRIPTION
**Pull request by hakbailey**

### CODE REVIEW

File: roles/connectivity_troubleshooter_igw/README.md [hunk 1]
```diff
@@ -1,22 +1,26 @@
-Role Name
+connectivity_troubleshooter_igw
 =========
 
 A role to troubleshoot connectivity issues between AWS resources in an Amazon VPC and an internet resource using an internet gateway.
 
+* Confirms whether the source has a public IP address associated with the destination resource.
+* Confirms whether the security group rules allow the needed traffic from the source to the destination resource.
+* Confirms whether the network ACLs allow the needed traffic from the source resource.
```
Comment (alinabuzachis) [inline roles/connectivity_troubleshooter_igw/README.md:8]: Not really sure if we should list these info here. The module eval_src_gateway_route module does that. I will probably suggest we generate the documentation for the module.

File: README.md [hunk 2]
```diff
@@ -12,15 +13,16 @@ Click on the name of a role to view that content's documentation:
...
-[cloud.aws_troubleshooting.connectivity_troubleshooter_validate](https://github.com/redhat-cop/cloud.aws_troubleshooting/blob/main/roles/awsconfig_multiregion_cloudtrail/README.md)|A role to validate input parameters for troubleshoot_connectivity_* roles and return connection next hop.
+[cloud.aws_troubleshooting.connectivity_troubleshooter](https://github.com/redhat-cop/cloud.aws_troubleshooting/blob/main/roles/connectivity_troubleshooter/README.md)|A role to troubleshoot connectivity issues between the following: a) AWS resources within an Amazon Virtual Private Cloud (Amazon VPC); b) AWS resources in different Amazon VPCs within the same AWS Region that are connected using VPC peering; c) AWS resources in an Amazon VPC and an internet resource using an internet gateway; d) AWS resources in an Amazon VPC and an internet resource using a network address translation (NAT) gateway.
```
Comment (alinabuzachis) [inline README.md:20]: I guess we should keep the description of the role short (it should be like a short description), fore more details, the user just need to access the README of the role itself (which should contain an exhaustive description).

File: roles/aws_setup_credentials/README.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
-# aws_setup_credentials
+aws_setup_credentials
+=====================
```
Comment (alinabuzachis) [inline roles/aws_setup_credentials/README.md:2]: These should be equivalent.

Comment (hakbailey) [reply roles/aws_setup_credentials/README.md:2]: Using two different types of heading styles was giving a Markdown linting error locally, we should pick one style and use it consistently within a document. So I updated this one to match the style used in the rest of the document.

File: roles/connectivity_troubleshooter_igw/README.md (hunk 1 above)
Comment (hakbailey) [reply roles/connectivity_troubleshooter_igw/README.md:8]: Per [the Jira ticket](https://issues.redhat.com/browse/ACA-1081), the point of this PR is to clarify what it is these roles are doing. These steps are ultimately what the role is doing (via the module) to identify potential sources of connectivity issues. I think the purpose of adding these here is that someone using the role shouldn't have to go into the module to get a general overview of how the role is troubleshooting connectivity.

File: README.md (hunk 2 above)
Comment (hakbailey) [reply README.md:20]: Sure, I didn't change the descriptions here, just the links, but if you have a suggestion for how to shorten this description while also keeping it clear that this role is essentially a combination of all the other roles I'm happy to update it!

**Review by GomathiselviS — APPROVED** [review]
There are a few ansible-lint issues that need to be addressed, but aside from that, everything looks good to me.

File: roles/connectivity_troubleshooter_igw/README.md (hunk 1 above)
Comment (abikouo) [reply roles/connectivity_troubleshooter_igw/README.md:8]: I would rather prefer keep this description here, the end user is supposed to know the internal design of the role so having the validation steps here in the documentation is helpful

**Review by abikouo — APPROVED** [review]
@hakbailey please note that #31 is fixing almost all of your ansible-lint errors

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**